modules/nf-core/last/mafconvert/tests/main.nf.test

```diff
@@ -0,0 +1,35 @@
+nextflow_process {
+
+    name "Test Process LAST_MAFCONVERT"
+    script "../main.nf"
+    process "LAST_MAFCONVERT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "last"
+    tag "last/mafconvert"
+
+    test("sarscov2 - bam") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'contigs.genome' ], // meta map
+                    file(params.test_data['sarscov2']['genome']['contigs_genome_maf_gz'], checkIfExists: true)
+                ]
+                input[1] = [ 'psl' ]
```

Comment:
You can just give strings like this without the square brackets btw.

Reply:
Is it easy to catch that the program has failed in that way, and force the exitcode to not be 0?

Reply:
I just tried `maf-convert` outside virtualisation and it exits with error status 1.  I do not know why it was 0 when run through Nextflow/SLURM/Singularity…

Reply:
Good catch!  I am all for setting pipefail, but isn't it already done at the fist line of `.command.sh`?
```
#!/bin/bash -euo pipefail
```

Reply:
Yeah, I think it should be set anyway, but we could easily split this to be a separate line